internal/pkg/deploy/cloudformation/env_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Package cloudformation provides functionality to deploy archer resources with AWS CloudFormation.
+// Package cloudformation provides functionality to deploy ECS resources with AWS CloudFormation.
```

Comment:
nit: maybe `"... to deploy CLI concepts with ..."`, just ECS feels like it's to deploy things regarding only Amazon ECS.